dist/index.html

```diff
@@ -61,14 +61,13 @@
         var key = encodeURIComponent($('#input_apiKey')[0].value);
         log("key: " + key);
         if(key && key.trim() != "") {
+            var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("api_key", key, "query");
+            window.swaggerUi.api.clientAuthorizations.add("api_key", apiKeyAuth);

```

Comment:
I suggest that, for backwards compatibility, that swagger-ui expose one global function to call the instance.  Otherwise, every swagger-ui deployment which doesn't update their index file (which is a typical upgrade scenario) will break.